src/templates/static/about/binary_in_numbers.jsx

```diff
@@ -21,19 +21,19 @@ const BinaryInNumbers = () => (
             <Numbers
                 className='client'
                 header={it.L('Active trading clients')}
-                text={it.L('Our active client base is expected to reach over 100,000 total clients this year.')}
+                text={it.L('Our active client base exceeded [_1] clients in [_2].', '127,000', '2019')}
                 image='chart-active-trading-clients'
             />
             <Numbers
                 className='transaction'
                 header={it.L('Number of transactions')}
-                text={it.L('We\'re projected to register over 197 million transactions this year, bringing the total number of contracts that were bought and sold on our platform since inception to over 789 million.')}
+                text={it.L('Our customers bought more than 275 million contracts on Binary.com in 2019, bringing the total number of contracts sold to over 1 billion since commencing business.')}
```

Comment:
please use a placeholder and it.website_name for binary.com

Reply:
should we use placeholders for the numbers and year here too?

Reply:
Last time you said inserting numbers without placeholders led to presentation issues. Except this, as numbers aren't dynamic here we can skip placeholders and insert them directly as literals.

Reply:
sure, but please update the binary.com placeholder :)